file_format.md

```diff
@@ -1,6 +1,10 @@
-# .simularium JSON Format
+# .simularium File Format
```

Comment:
the file format is no longer just JSON, update descriptions of structure and files